lxd/storage/backend_lxd.go

```diff
@@ -7090,9 +7091,32 @@ func (b *lxdBackend) GenerateInstanceBackupConfig(inst instance.Instance, snapsh
 				})
 			}
 		}
+
+		for _, device := range inst.ExpandedDevices() {
+			if device["type"] != "disk" || instancetype.IsRootDiskDevice(device) {
```

Comment:
later you have similar logic but it also contains a pool name check:

```go
device["pool"] == ""
```

shouldnt this also have this check?

Reply:
possibly you have a commit that has the old code included in it accidentally and then it gets modified later?